python_modules/dagster/dagster_tests/logging_tests/test_log_manager.py

```diff
@@ -9,33 +10,28 @@
 from dagster._core.execution.plan.objects import ErrorSource, StepFailureData
 from dagster._core.execution.plan.outputs import StepOutputData, StepOutputHandle
 from dagster._core.log_manager import (
-    DagsterLoggingMetadata,
     DagsterLogHandler,
+    DagsterLogHandlerMetadata,
     DagsterLogManager,
-    DagsterMessageProps,
-    construct_log_string,
+    DagsterLogRecordMetadata,
+    construct_log_record_message,
 )
-from dagster._utils.error import serializable_error_info_from_exc_info
+from dagster._utils.error import SerializableErrorInfo, serializable_error_info_from_exc_info
 
 
-def test_metadata_event_tags():
-    logging_metadata = DagsterLoggingMetadata(
-        run_id="f79a8a93-27f1-41b5-b465-b35d0809b26d",
-        job_name="my_job",
-        job_tags={"foo": "bar"},
-    )
-
-    all_tags = logging_metadata.all_tags()
-    event_tags = logging_metadata.event_tags()
+def _construct_log_handler_metadata(**kwargs) -> DagsterLogRecordMetadata:
+    all_keys = DagsterLogHandlerMetadata.__annotations__.keys()
+    defaults = {k: None for k in all_keys if k not in kwargs}
+    return DagsterLogHandlerMetadata(**kwargs, **defaults)  # type: ignore
 
-    assert all_tags["job_name"] == "my_job"
-    assert all_tags["job_tags"] == "{'foo': 'bar'}"
 
-    assert event_tags["job_name"] == "my_job"
-    assert "job_tags" not in event_tags
```

Comment:
Deleted this test because these accessors on `DagsterLoggingMetadata` (now `DagsterLogHandlerMetadata`) no longer exist. They were only called once in the `StepExecutionContext` to obtain `tags`/`event_tags`, and the requisite logic is now in the `StepExecutionContext` properties.